<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 85, 139);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="booklistB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);
</string>
    </property>
    <property name="text">
     <string>စာအုပ်စာရင်း</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Library Management System Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addbookB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>စာအုပ်ထပ်ထည့်ရန်</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addusrB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>စာအုပ်ငှားသူထပ်ထည့်ရန်</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bookbowB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>စာအုပ်ငှားယူမှုစာရင်း</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bookloB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>စာအုပ်နေရာ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pyidaungsu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 85, 139);
border-radius: 20px;
border : 2px solid rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>booklistB</zorder>
   <zorder>label</zorder>
   <zorder>addbookB</zorder>
   <zorder>addusrB</zorder>
   <zorder>bookbowB</zorder>
   <zorder>bookloB</zorder>
   <zorder>aboutB</zorder>
   <zorder>l1</zorder>
   <zorder>l2</zorder>
   <zorder>l3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
